controllers/application/deployment.go

```diff
@@ -52,6 +58,12 @@ func (r *ApplicationReconciler) reconcileDeployment(ctx context.Context, applica
 			"prometheus.io/scrape":            "true",
 		}
 
+		// add an external link to argocd
+		ingresses := application.Spec.Ingresses
+		if len(ingresses) > 0 {
+			deployment.ObjectMeta.Annotations[AnnotationKeyLinkPrefix] = fmt.Sprintf("https://%s", ingresses[0])
```

Comment:
This will fail if the Annotations map is a nil map. I'm not sure this will happen, but its worth having a check for it. Check out how its done with Application labels here: https://github.com/kartverket/skiperator/blob/main/pkg/util/reconciler.go#L143

Reply:
It's explicitly set on line 56, so it can't be nil. That's because this PR was updated against `main`, which introduces the argocd annotation.

Reply:
aha, i missed that :)